src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -13,9 +13,11 @@ import ArrowButton from "../../../../components/Buttons/ArrowButton";
 import { isDigital } from "../../utils/helpers";
 import { listId, ListType } from "../../../../core/utils/types/list-type";
 import SelectableMaterialSkeleton from "./selectable-material-skeleton";
+import { ILLBibliographicRecord } from "../../../../core/fbs/model";
 
 interface SelectableMaterialProps {
   identifier?: string | null;
+  ilBibliographicRecord?: ILLBibliographicRecord;
```

Comment:
```suggestion
  illBibliographicRecord?: ILLBibliographicRecord;
```
And the same for remaining `ilBibliographicRecord`´s